### ISSUE TITLE
Choose which media to share when sharing an album

### ISSUE THREAD

**Issue by xgaia** [photoview/photoview#694]
**Is your feature request related to a problem? Please describe.**

The sharing functionality share all the pictures of an album. It well be great if we can share a selection of pictures on an album (the favorites one).

**Describe the solution you'd like**

When I share an album, I would like to share only the favorite pictures. On the Sharing menu, an option to share only favorites pictures would be great.

**Comment by kkovaletp** [comment]
@xgaia, thanks for your feature request.

Sharing only previously selected media from the album, when you share that album, might be an interesting feature. However, using favorites for this wouldn't be the best solution, as these are 2 different purposes, and mixing them together wouldn't work for most users, as well as it will create confusion of unexpected behavior.

So, I'd propose adding a possibility to multi-select (unselect not needed, select and unselect all, select only needed) media on the album. By default, all media should be selected when a user starts the process of sharing the media.
The corner case and potentially a problem for implementation of this functionality is what to do with sub-albums and media, located inside of them. There might be a hierarchy of any complexity level. Maybe we can show a list of paths to each media of the current album and all its sub-albums, but in this case, it would be difficult to understand what they are. If we show the default view with a preview of each media, there won't be much space for its path.
In other words, we need to design a better solution here from the UX PoV.

**Comment by viktorstrate** [comment]
This is also mentioned in #414 which proposes a larger and more concrete solution to this problem?

**Comment by kkovaletp** [comment]
Comparing this use case with the one, discussed in the #414, it looks like the solution for that issue should solve this one as well, so I'm closing this issue as a duplicate
